src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodash from 'lodash';
```

Comment:
lets not load the entire lodash library, only the functions you need
```suggestion
import cloneDeep from 'lodash/cloneDeep';
```